storage/src/vespa/storage/config/stor-distributormanager.def

```diff
@@ -286,3 +286,9 @@ num_distributor_stripes int default=0 restart
 ## bucket due to being blocked by concurrent operations. This avoids potential head-of-line
 ## blocking of later buckets in the priority database.
 implicitly_clear_bucket_priority_on_schedule bool default=false
+
+## Enables sending merges that are not forwarded between content nodes in strictly
```

Comment:
Consider clarifying the first sentence: "Enables sending merges that are forwarded between content nodes in ideal state node key order, instead of strictly increasing node key order (which is the default)"

Reply:
Done